doc/resources/resources.md

```diff
@@ -0,0 +1,7 @@
+# Resources
+
+```{toctree}
+Flame API        <https://pub.dev/documentation/flame/latest/>
```

Comment:
I wonder if it's possible to link to the API docs which correspond to the docs version in the top menu?

Reply:
The docs are being published by this script: https://github.com/flame-engine/flame-docs-site/blob/main/publish.sh, and there are certain replacements being secretly done. Say, a string "<VERSION>" will get replaced by the corresponding docs version. Though I'm not sure if it'll be in the right format to use in a URL, but if not the publishing script can be modified to add a special rule for this file.

Reply:
It is the correct string in that variable since the API docs are just using the plain version number (without a v prefix) https://pub.dev/documentation/flame/1.3.0/

Reply:
Well you would need to use JS to get the current selected version because the user is switching that entirely at runtime, which can be done with: `getCurrentDocVersion()`.  BUT does toctree support that?  I don't know.  That script only gets executed during build right or are the pages dynamic server content not static html?

Reply:
When they switch at runtime, a new page with a new URL gets loaded. That page was generated by the build script at some point, and the build script should have made the substitutions according to the page's version. So it should work (in theory), at least it's worth a try.

Reply:
Right, but we need the version that the user selects at runtime not generating a link in a toctree at build time.  I think a trick could be to amend the existing JS builder to also build the API link based on the version.  I will give this a go in a bit.

Reply:
It generates one page per version, so it should still not be a problem with just setting it to `<VERSION>`, try to just set it to that and see if it works. :)

Reply:
Oh!  Ok, I completely misunderstood the build process then.  I am dense. Apologies.  Let me do that now.

Reply:
So that builds but does not work.  With <VERSION> in the toctree in *.md file BUT inside of a link directive "<https://xxxx/<VERSION>/" it just fails to even include that link now in the sidebar.  I will keep playing with this.

Reply:
I could not get it to recognize `<VERSION>` for the pre_processor to work, BUT I did get it working with some jQuery.  Some basically whatever version the user chooses, it just changes the link dynamically at runtime.  If someone can figure out how to get `<VERSION>` in the `resources.md` file as mentioned above, I can convert it over, otherwise, this is working locally.

Reply:
That is what I suspected, but without a way to test, isn't that an issue for other people working on docs too?  The jQuery solution does the same thing (achieves the same results I should say) but works in local and should work in prod.  The only downside is if someone had JS disabled, but then again, all the docs would stop working.

Reply:
I still think that we should go with the `<version>` placeholder instead of JS, it's much cleaner. You can test it locally by building the docs in the same way as the doc action is building them.

Reply:
We could probably do that.  Just come up with a new keyword that doesn't require `<>`.  That is reserved in toctree and I tried escaping it, but it never was interpreted correctly by SED.  Let me play with this a bit.

Reply:
Great, you have found this repository too right where our publish script is (and where we do the replacements)?
https://github.com/flame-engine/flame-docs-site

Reply:
Yes

Reply:
Ok, I have added a PR: https://github.com/flame-engine/flame-docs-site/pull/9

I have spent hours trying to use extlinks and every other extension under the sun to do this within MyST and Sphinx itself.  Ultimately, I have read numerous StackOverflow posts that just really seem to indicate you can't do markdown in toctree's.  Extlinks extension fails in the toctree but worked awesome in a regular document.  Anyway, I think those sed commands will get the job done and I restricted it to the specific `resources.md` file so there isn't any accidental bleed over.  Very interested in your thoughts on this.

Reply:
Updated to support one format.